src/WpfMath/Fonts/WpfSystemFont.cs

```diff
@@ -155,6 +155,6 @@ private static TeXFontMetrics GetFontMetrics(char c, Typeface typeface)
         var scalingFactor = typeface.FontFamily.LineSpacing / typeface.FontFamily.Baseline;
         height /= scalingFactor;
 
-        return new TeXFontMetrics(formattedText.Width, height, depth, 0, 1.0);
```

Comment:
Why have you reverted this? It seems wrong: it will stop preserving the "depth" of the system font characters. Example:
![image](https://github.com/ForNeVeR/xaml-math/assets/92793/8cc72aa6-b096-4b16-9fd6-87841f302648)
(see how the selection box is wrong now)

Reply:
Your solution doesn't solve the problem. In addition, it breaks other types of matrices.
![image](https://github.com/ForNeVeR/xaml-math/assets/63106764/91c68524-f5a9-439c-9ee0-3f81760203fe)
![image](https://github.com/ForNeVeR/xaml-math/assets/63106764/1b262cc8-269c-4123-9fe0-9e61f3389f68)


> Why have you reverted this? It seems wrong: it will stop preserving the "depth" of the system font characters.

Because the letters overhang is under the baseline. If an alignment by the baseline, then the overhang must be below the baseline (seems logic). Otherwise a characters will be above the required line (in this case will be need substract depth at boxes placing - it is also seems logic 🤔).